### PR TITLE
test notifications

### PR DESCRIPTION
**Pull request by TDM1911**
## Related Tickets:
- [#31885](https://edu-redmine.sun-asterisk.vn/issues/31885)

## WHAT this PR does?
- Test NotifyEvent.
- Test ProjectApproved, Warning notifications.

## Notes Impacted Areas:
- N/A.

## Attachments:
ảnh
ảnh

### CODE REVIEW

**Comment by TDM1911** [comment]
ready

**Review by kienpham9819 — APPROVED** [review]

File: tests/Unit/Notifications/ProjectApprovedTest.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        unset($this->notification);
+        unset($this->projectName);
+        unset($this->url);
+        parent::tearDown();
+    }
+
+    public function test_via_function()
+    {
+        $this->assertEquals(['database'], $this->notification->via('App\Models\User'));
```
Comment (huyenttm-0901) [inline tests/Unit/Notifications/ProjectApprovedTest.php]:
```suggestion
        $this->assertEquals(['database'], $this->notification->via(User::class));
```

File: tests/Unit/Notifications/WarningTest.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public function tearDown() : void
+    {
+        unset($this->notification);
+        unset($this->data);
+        parent::tearDown();
+    }
+
+    public function test_via_function()
+    {
+        $this->assertEquals(['database'], $this->notification->via('App\Models\User'));
```
Comment (huyenttm-0901) [inline tests/Unit/Notifications/WarningTest.php]: tương tự

**Comment by TDM1911** [comment]
ready

**Pull request merged**